quartical/config/argument_schema.yaml

```diff
@@ -306,6 +306,14 @@ solver:
         the solver.iter_recipe loops through the chain multiple times. The
         reweighting step only happens once per traversal of the chain.
 
+    weight_flag_threshold:
+      dtype: float
+      default: 0.0
```

Comment:
I cannot recall if the stimelation branch preserved the notion of a parameter being `Optional[Type]`. This is the kind of parameter that could be `None` unless specified, but it isn't a big deal.

Reply:
I believe this notion is still in place but then we would have to deal with the ```None``` case separately eg. this
https://github.com/ratt-ru/QuartiCal/blob/63702764e2e60900cecafedd7e35a876973afa1e/quartical/calibration/solver.py#L153
could be something like
```
solver_opts.reweighting_flag_threshold if solver_opts.reweighting_flag_threshold is not None else 0
```
Would that be preferred?

Reply:
Leave it as is for now, I think. Can always fiddle with internal stuff like this later.